spec.emu

```diff
@@ -1006,6 +1006,22 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
         BindingIdentifier[~Yield, +Await]
     </emu-grammar>
 
+    <emu-clause id="sec-imports-static-semantics-early-errors">
+      <h1>Static Semantics: Early Errors</h1>
+      <emu-grammar>ModuleItem : ImportDeclaration</emu-grammar>
+      <ul>
+        <li>
+          It is a Syntax Error if the BoundNames of |ImportDeclaration| contains any duplicate entries.
+        </li>
+      </ul>
+      <emu-grammar><ins>ImportDeclaration : `import` `source` ImportedBinding FromClause `;`</ins></emu-grammar>
+      <ul>
+        <li>
+          <ins>It is a Syntax Error if the source text matched by |ImportedBinding| is `"from"`.</ins>
```

Comment:
I wonder if we should instead use:

```
It is a Syntax Error if the sole [BoundName](https://tc39.es/ecma262/#sec-static-semantics-boundnames) of |ImportedBinding| is "from".
```

Reply:
This would ban `import from from "specifier"`, right?

Reply:
Meant it more as an editorial change - wouldn't apply to `import from from 'specifier'` as it would still only be for this production I think?